src/pages/portfolio/index.tsx

```diff
@@ -75,27 +76,75 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+const numberOfMaxTabs = 3;
+const OtherLabel = 'More';
+
+type GroupedProjectsByLabelType = {
+  projectLabel: string;
+  projectProperties: ProjectType[];
+};
+
+const compareLengthOfProjects = (
+  longerLengthGroupedProjectsByLabel: GroupedProjectsByLabelType,
+  shorterLengthGroupedProjectsByLabel: GroupedProjectsByLabelType,
+): number => {
+  return (
+    shorterLengthGroupedProjectsByLabel.projectProperties.length -
+    longerLengthGroupedProjectsByLabel.projectProperties.length
+  );
+};
+
+const getTabsData = (projects: ProjectType[]) => {
+  const groupedProjects: Record<string, ProjectType[]> = _.groupBy(projects, 'projectLabel');
```

Comment:
We could add `lodash` if it was useful application-wide, for a single-use case you don't need a whole library.
Moreover, in my opinion, you don't need a generic function for only handling project labels.

Let's define our own groupByProjectLabel and move it to `/views/portfolio-page/utils.ts`

If you want you can try this, it should work 😄 
I took a generic groupBy method and transformed it to our use-case, lodash bundle size is sooo huge 🍭 
```typescript
function groupByProjectLabel(projects: ProjectType[]): Record<string, ProjectType[]> {
  return projects.reduce((groupedProjects, project) => {
    // eslint-disable-next-line no-param-reassign
    (groupedProjects[project.projectLabel] = groupedProjects[project.projectLabel] || []).push(project);
    return groupedProjects;
  }, {} as { [key: string]: ProjectType[] });
}
```